transmogrifier.js

```diff
@@ -413,6 +413,9 @@ const filters =
     },
     to_lower: async function (data, params) {
         return data.toLowerCase()
+    },
+    oden_to_transmogrifier: async function (data, params) {
```

Comment:
doesn't need to be a builtin, odenToTransmog can be a filter in the filters repo and we can pull it that way